must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -100,6 +103,7 @@ for ns in $namespaces; do
                 { timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "${ceph_commands[$i]} --connect-timeout=15" >> "${COMMAND_OUTPUT_FILE}"; } >> "${BASE_COLLECTION_PATH}"/gather-debug.log 2>&1
                 { timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "${ceph_commands[$i]} --connect-timeout=15 --format json-pretty" >> "${JSON_COMMAND_OUTPUT_FILE}"; } >> "${BASE_COLLECTION_PATH}"/gather-debug.log 2>&1
             done
+            for i in `timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "ceph osd lspools --connect-timeout=15"|awk '{print $2}'`; do { timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd ls -p $i" >> ${COMMAND_OUTPUT_DIR}/pools_rbd_$i; } >> "${BASE_COLLECTION_PATH}"/gather-debug.log 2>&1; done
```

Comment:
Why are you running the `ceph osd lspools` command again? Can't you just read the command output file?

Reply:
I was unable to directly  consume the command output file. This way it was less code of line and variables. 
e.g. while read line; do rbd ls -p $line |awk '{print $2}' ; done < ceph_osd_pools
 the awk command doesn't work like this. if we want to read the output file I can find the alternative for it.